platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/gossip/config/GossipConfig.java

```diff
@@ -24,7 +24,7 @@
 
 @ConfigData("gossip")
 public record GossipConfig(
-        @ConfigProperty(defaultValue = Configuration.EMPTY_LIST) List<NetworkEndpoint> networkEndpoints,
+        @ConfigProperty(defaultValue = Configuration.EMPTY_LIST) List<NetworkEndpoint> interfaceBindings,
```

Comment:
Could be add  class level javadoc explaining the the meaning and purpose of each parameter?